geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/CacheClientNotifierTest.java

```diff
@@ -140,6 +142,35 @@ public void eventsInClientRegistrationQueueAreSentToClientAfterRegistrationIsCom
     verify(cacheClientProxy, times(1)).deliverMessage(isA(HAEventWrapper.class));
   }
 
+  @Test
+  public void clientRegistrationFailsQueueStillDrained()
```

Comment:
Hi Ryan, just wanted to know that the test mentions that clientRegistrationFails**QueueStillDrains**. Is there an assert needed to make sure that the queue actually drained? the last assert is that an IOException is thrown

Reply:
You are right that this test is not really complete.  I think initially I was punting on that validation because the registration queue manager is not publicly accessible.  I am going to make a commit so we can inject and verify these calls on the queue manager.